adapters/Chartboost/ChartboostAdapter/GADMAdapterChartboostSingleton.m

```diff
@@ -35,6 +35,9 @@ @interface GADMAdapterChartboostSingleton () <ChartboostDelegate> {
   NSMutableArray<ChartboostInitCompletionHandler> *_completionHandlers;
 }
 
+@property (nonatomic) NSMapTable<CHBBanner *, id<CHBBannerDelegate>> *bannersToDelegates;
+@property (nonatomic) NSMutableArray<CHBBanner *> *loadingBanners;
```

Comment:
Does Chartboost support loading multiple banner ads using the same location?

If so, then I think we can remove this, since I don't see this array getting checked anywhere else.

Reply:
We do support multiple banners with the same location.

This array is still needed though, please check out the didCacheAd:error: implementation (it was moved to GADMAdapterChartboostBanner.m).
We use it to keep the Chartboost banner objects in memory while they are being loaded.